### ISSUE TITLE
#a11y: account menu trigger always announced as "expanded", underlying page still exposed to AT

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/desktop#1348]
### Steps To Reproduce

1. Using a screen reader, set focus to the new account menu trigger (note that it's announced as "expanded" even when it's not)
2. Open the menu; it effectively acts as a modal, preventing any tabbing out
3. Use the screen reader's reading keys (generally, cursor keys)

### Expected Result

for 1., the control should be announced as "collapsed" (i.e. `aria-expanded="false"`), not expanded (`aria-expanded="true"`). in fact, if it behaves less like an expand/collapse control, but more like a traditional menu (which is the case here since the resulting menu is "modal" in essence), instead of announcing as expanded or collapsed, it should announce as having a menu/dropdown (i.e. use `aria-haspopup="true"` rather than `aria-expanded="..."`). EDIT: ah just realised, as it already announces as "subMenu", I suspect the `aria-haspopup="true"` is already there...so in essence just "remove the unnecessary and incorrect `aria-expanded="..."` attribute altogether.

for 3. if it's truly acting like a modal, then the underlying page itself should be hidden from assistive technologies as well - a user should not be able to move the reading cursor back out of it. consider using `aria-hidden="true"` on the underlying page/content, or something like the `inert` attribute and polyfill https://github.com/WICG/inert

### Actual Result

for 1. the control is incorrectly announced by assistive technologies as being "expanded" when it clearly isn't.

for 3. using reading keys an AT user can navigate right back into the underlying page/content.

### Screenshots or Videos

Video using NVDA. `Tab` to the account menu trigger (note how it announces as "expanded"). Once in the open menu, using the up arrow/cursor to read in reverse back out of the menu and through the underlying page/content

https://user-images.githubusercontent.com/895831/154139237-38cb1fad-44c3-4a2d-82df-965a507e2aa2.mp4

### Operating System

Windows

### Operating System Version

10

### Installation method

Direct Download (from bitwarden.com)

### Build Version

Version 1.31.3 Shell 16.0.7 Renderer 96.0.4664.110 Node 16.9.1 Architecture x64

**Comment by cscharf** [comment]
Thank you @patrickhlauke , will get this over to our product and design team, much appreciated!
